app/src/lib/git/stash.ts

```diff
@@ -144,23 +144,62 @@ function extractBranchFromMessage(message: string): string | null {
   return match === null || match[1].length === 0 ? null : match[1]
 }
 
-/** Get the files that were changed in the given stash commit.
- *  This is different than `getChangedFiles` because stashes
- *  have _3 parents(!!!)_
+/**
+ * The SHA for the [empty tree](https://stackoverflow.com/questions/9765453/is-gits-semi-secret-empty-tree-object-reliable-and-why-is-there-not-a-symbolic).
+ *
+ */
+const NullTreeSHA = '4b825dc642cb6eb9a060e54bf8d69288fbee4904'
```

Comment:
Maybe it could be worth putting the “Magic” string into Dugite?

Reply:
yeah i think that's a good idea @say25!